src/server/replica.h

```diff
@@ -267,6 +269,8 @@ class Replica {
   std::optional<base::IoBuf> leftover_buf_;
   std::unique_ptr<facade::RedisParser> parser_;
   facade::RespVec resp_args_;
+  base::IoBuf resp_buf_;
```

Comment:
could you use the leftover_buf_?

Reply:
Possibly, but I reset the buffer each SendCommand, and I prefer not to interfere with the state that's kept in this buffer since it's handled a bit complicated.